letsencrypt/build.yaml

```
@@ -18,3 +18,4 @@ args:
   CERTBOT_NETCUP_VERSION: 1.1.1
   CERTBOT_NJALLA_VERSION: 1.0.0
   CERTBOT_VERSION: 1.21.0
+  CERTBOT_DNS_IONOS_VERSION: 2022.5.15
```

```suggestion
  CERTBOT_DNS_IONOS_VERSION: 2022.11.24
```
Is there a particular reason we aren't picking the current latest version here?